frontend/src/app/person/person.component.spec.ts

```diff
@@ -42,7 +42,7 @@ describe('PersonComponent', () => {
     abroadFamilySituation: null
   };
 
-  const activatedRoute = {
+  const activatedRoute: any = {
```

Comment:
nitpicking `const activatedRoute = { ... } as ActivatedRoute;` would be better IMO